engine/pom.xml

```diff
@@ -206,7 +208,6 @@
                             <classesDirectory>${project.build.testOutputDirectory}</classesDirectory>
                             <includes>
                                 <include>org/hibernate/validator/testutil/**</include>
-                                <include>org/hibernate/validator/test/internal/metadata/provider/AnnotationMetaDataProviderTestBase*</include>
```

Comment:
No, I bit of a mistake from my part. I already changed this when merging your pull request. I just forgot to add this as well. I should have just staged this hunk. In the merged code I got rid of the base class and copied the few helpers needed. I think just creating the 'testing' jar from `org/hibernate/validator/testutil` is nicer. In particular if we think about making this its own module.